### PR TITLE
added .gitattributes file for Unix-like end of line

### PR DESCRIPTION
**Pull request by simonjenga**
#92 added .gitattributes file for Unix-like end of line

### CODE REVIEW

**Comment by dmarkov** [comment]
@simonjenga Many thanks, I will find someone to review it before we merge

**Comment by dmarkov** [comment]
@caarlos0 please review this one

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Check out all text files in UNIX format.
+* text eol=lf
+
+# Explicitly declare text files we want to always be normalized and converted
```
Comment (caarlos0) [inline .gitattributes]: @simonjenga I don't thind the lines bellow are needed... git already know what is and what is not a text file...

**Comment by simonjenga** [comment]
@caarlos0 I have removed the unnecessary lines in .gitattributes

**Comment by caarlos0** [comment]
@simonjenga thanks!

**Comment by caarlos0** [comment]
@rultor good to merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by caarlos0** [comment]
@yegor256 didn't get paid here

**Comment by caarlos0** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@dmarkov pls add extra 15 mins to @caarlos0

**Comment by caarlos0** [comment]
@yegor256 didn't get paid for this one too

**Pull request merged**